app/models/featured_tag.rb

```diff
@@ -30,6 +30,8 @@ class FeaturedTag < ApplicationRecord
 
   scope :by_name, ->(name) { joins(:tag).where(tag: { name: HashtagNormalizer.new.normalize(name) }) }
 
+  delegate :formatted_name, to: :tag
```

Comment:
This is needed because the settings helper access is actually handling a collection of `FeaturedTag` instances.

We could alternately remove this, and adjust that usage so that it operates on the tags of those featured tags.